src/compare/compare.py

```diff
@@ -78,8 +78,17 @@ def _setup_plugins(self):
     def _init_plugins(self):
         self.source = import_plugins('compare.plugins', 'plugins/compare')  # pylint: disable=attribute-defined-outside-init
         for plugin_name in self.source.list_plugins():
-            plugin = self.source.load_plugin(plugin_name)
-            plugin.ComparePlugin(self, config=self.config, db_interface=self.db_interface)
+            try:
+                plugin = self.source.load_plugin(plugin_name)
+            except Exception as e:
+                # This exception could be caused by invalid syntax
+                # Invalid syntax may be caused by plugin dependencies that got
+                # upgraded to incompatible versions
```

Comment:
The dependencies could also simply be missing (either if new dependencies were added and the user did not re-run the installation or if the user didn't install FACT correctly in the first place)